samples/client/petstore/dart/flutter_petstore/openapi/docs/PetApi.md

```diff
@@ -243,21 +245,21 @@ Name | Type | Description  | Notes
 [[Back to top]](#) [[Back to API list]](../README.md#documentation-for-api-endpoints) [[Back to Model list]](../README.md#documentation-for-models) [[Back to README]](../README.md)
 
 # **updatePet**
-> updatePet(body)
+> updatePet(pet)
```

Comment:
@pmundt Which version of OpenAPI Generator are you using when working on this PR?

If it's the latest version, `body` should be used instead of `pet`.

Reply:
This was done with the current master from git shortly before the PR was created - with a full clean/install. I can attempt to wipe the tree and try it again in case this has accidentally cached some old build artifacts.

Reply:
👌 let me pull the changes locally to test it out.

Reply:
Your change looks good. I'll troubleshoot after merging your fix into master.